content/applications/sales/point_of_sale/payment_methods/terminals/viva_wallet.rst

```diff
@@ -0,0 +1,159 @@
+===========
+Viva Wallet
+===========
+
+Connecting a **Viva Wallet payment terminal**  allows you to offer a fluid payment flow to your
+customers and ease the work of your cashiers.
+
+.. note::
+   Viva Wallet lets you turn your phone into a mobile card reader: `Tap On Phone
+   <https://www.vivawallet.com/gb_en/blog-tap-on-phone-gb>`_.
+
+Configuration
+=============
+
+Start by creating your Viva Wallet account on `Viva Wallet's website <https://www.vivawallet.com>`_.
```

Comment:
I was not able to finalize the account creation to test because I was missing an enterprise number. How can I do?